endpoints-framework/src/main/java/com/google/api/server/spi/config/model/SchemaRepository.java

```diff
@@ -41,8 +44,19 @@
       .setType("object")
       .build();
 
+  private static final EnumSet<FieldType> SUPPORTED_MAP_KEY_TYPES = EnumSet.of(
+          FieldType.STRING,
+          FieldType.ENUM,
+          FieldType.BOOLEAN,
+          FieldType.INT8, FieldType.INT16, FieldType.INT32, FieldType.INT64,
+          FieldType.FLOAT, FieldType.DOUBLE,
+          FieldType.DATE, FieldType.DATE_TIME
```

Comment:
I think it could be cleaner to move this logic into the FieldType enum by adding a new field :
````
private final boolean serializeableIntoString; // or scalarValue?
````

Reply:
I actually prefer the current solution: the list of types usable as Map keys is somehow arbitrary, and might evolve over time. It is not intrinsic to the FieldTypes, on the contrary of the other existing fields.

Reply:
ok